### ISSUE TITLE
Go client: NetworkIpRef requires Kind property which doesn't exist in v1alpha.ResourceRef type

### ISSUE THREAD

**Issue by averzicco** [GoogleCloudPlatform/k8s-config-connector#675]
### Checklist

- [X] I did not find a related open issue.
- [X] I did not find a solution in the troubleshooting guide: (https://cloud.google.com/config-connector/docs/troubleshooting)
- [X] If this issue is time-sensitive, I have submitted a corresponding issue with [GCP support](https://cloud.google.com/support-hub).

### Bug Description

[NetworkIpRef](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/e621d3cfcf15188e2d353dbbfe87cde851e58d82/pkg/clients/generated/apis/compute/v1beta1/computeinstance_types.go#L226) is [v1alpha.ResourceRef](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/e621d3cfcf15188e2d353dbbfe87cde851e58d82/pkg/clients/generated/apis/k8s/v1alpha1/types.go#L59-L66) type but the [CRD for ComputeInstance](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/e621d3cfcf15188e2d353dbbfe87cde851e58d82/config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_computeinstances.compute.cnrm.cloud.google.com.yaml#L571-L603) requires also the Kind property, which is not part of v1alpha1.ResourceRef, this makes impossible to create ComputeInstance resource when it needs a ComputeAddress reference.

Validation fails with:
Invalid value: "":"spec.networkInterface.networkIpRef" must validate one and only one schema (oneOf). Found none valid, spec.networkInterface.networkIpRef.kind: Required value

Tested on v1.89.0 of the go client release

### Additional Diagnostic Information

```
NetworkIpRef: &v1alpha1.ResourceRef{
	Name:      "compute-address",
	Namespace: "namespace",
},
```

### Kubernetes Cluster Version

Client Version: v1.20.1 Server Version: v1.23.5-gke.2400

### Config Connector Version

v1.89.0

### Config Connector Mode

namespaced mode (default)

### Log Output

Invalid value: "":"spec.networkInterface.networkIpRef" must validate one and only one schema (oneOf). Found none valid, spec.networkInterface.networkIpRef.kind: Required value

### Steps to reproduce the issue

create a ComputeInstance object which refers a ComputeAddress in the networkInterface with go-client release v1.89.0 (latest available)

**Comment by caieo** [comment]
@averzicco, we recognize that this is a bug in our go-clients and have added it as a bug to fix in the future. We apologize for any inconvenience this brings you.

**Comment by caieo** [comment]
@averzicco, this should now be fixed as of v1.92.0. Please give it a try! I will close this issue for now, but feel free to open another issue if you find one.
